website/docs/guides/migrating-from-v2-to-v3.mdx

```diff
@@ -0,0 +1,87 @@
+# Migrating from `v2` to `v3`
+
+With new major version comes breaking changes. This section will guide you through the process of migrating from `v2` to `v3`.
+
+### 1. Remove `graphql` as a peer dependency
+
+We have designed the new `envelop` to be engine agnostic. This allows you to use any GraphQL engine you want. This means that `graphql` is no longer a peer dependency and `envelop` simply just wraps the `parse`, `validate`, `execute` and `subscribe` functions that you provide.
+
+```diff
+import { envelop } from '@envelop/core';
++ import { parse, validate, execute, subscribe } from 'graphql';
+
+- const getEnveloped = envelop([ ... ])
++ const getEnveloped = envelop({ parse, validate, execute, subscribe, plugins: [ ... ] })
+```
+
+### 2. Remove `useAsyncSchema` plugin
+
+This was a mistake from beginning as we cannot asynchronously `validate` and `parse` since with [`graphql`](https://github.com/graphql/graphql-js) these functions are synchronous in nature.
```

Comment:
misses documentation path for correctly using an async loaded schema

Reply:
Confused? We removed async loaded schema cause it was never “async” in nature

Reply:
```
const schema = await buildSchem()

const envelop = getEnveloped({plugins: useSchema(schema)})
```

I meant sth like this 😅